src/Microsoft.AspNet.Mvc.Core/DisableCorsAuthorizationFilter.cs

```diff
@@ -0,0 +1,36 @@
+// Copyright (c) Microsoft Open Technologies, Inc. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Threading.Tasks;
+using Microsoft.AspNet.Cors.Core;
+using Microsoft.AspNet.WebUtilities;
+using Microsoft.Framework.Internal;
+
+namespace Microsoft.AspNet.Mvc
+{
+    /// <summary>
+    /// An <see cref="ICorsAuthorizationFilter"/> which ensures that an action does not run for a pre-flight request.
+    /// </summary>
+    public class DisableCorsAuthorizationFilter : ICorsAuthorizationFilter
+    {
+        /// <inheritdoc />
+        public Task OnAuthorizationAsync([NotNull] AuthorizationContext context)
```

Comment:
Because the enable cors is async, this one does not have to be though, but I do not want multiple interfaces.